Build/Common.Build.Default.props

```diff
@@ -29,8 +29,9 @@
 
   <!-- Common build settings -->
   <PropertyGroup>
-    <OptimizedBuild />
-    <OptimizedBuild Condition="'$(Configuration)'=='Release' or '$(Configuration)'=='Test'">true</OptimizedBuild>
+    <FreeBuild />
+    <FreeBuild Condition="'$(Configuration)'=='Release' or '$(Configuration)'=='Test'">true</FreeBuild>
```

Comment:
Or just change all reference of '$(FreeBuild)'!='true' to '$(Configuration)'=='Debug' and vice versa.